src/pages/PostAJob.js

```diff
@@ -112,7 +180,7 @@ const PostAJob = () => {
                 data-cy='edit-job-button'
                 className='flex items-center mb-3 text-teal-600 text-lg font-bold'
                 onClick={(e) => {
-                  setStatus(1)
+                  history.push('/post-a-job?s=1')
```

Comment:
Out of curiousity, what is the `s=1` query parameter?

Reply:
S = status. There's 3 stages of posting a job and we needed a query param in the url to be able to direct users to specific points in the process.